pages/portfolio.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Jan's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Lora:400,400i|Open+Sans:300,400,600i,800" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Amatic+SC" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/normalize.css">
+    <link rel="stylesheet" href="../styles/main.css" type="text/css" media="screen">
+  </head>
+
+  <body>
+    <header>
+      <h2><a href="../index.html">Jan Edrozo</a></h2>
+      <nav>
+        <ul>
+          <li><a href="about.html">About</a></li>
+          <li><a href="portfolio.html">Portfolio</a></li>
+          <li><a href="blog.html">Blog</a></li>
+          <li><a href="mailto:jedrozo@gmail.com?Subject=Oh%20Hai%20There" target="_top">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="portfolio">
+        <img class="center-piece" alt="portfolio in cursive" src="../images/portfolio_cursive.png">
+      </section>
+
+      <section class="main-text">
+        <h2>Recent Projects</h2>
+        <hr />
+        <p>
```

Comment:
If there were more content here, I might have put each "project" in a separate container that represented one "project" each